src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -288,6 +289,115 @@ public void SetThreadPrincipal(IPrincipal principal)
             }
         }
 
+        public ObjectHandle CreateInstance(string assemblyName, string typeName)
+        { 
+            if (assemblyName == null)
+                throw new ArgumentNullException(nameof(assemblyName));
+
+            return Activator.CreateInstance(assemblyName, typeName);
+        }
+
+        public ObjectHandle CreateInstance(string assemblyName, string typeName, bool ignoreCase, BindingFlags bindingAttr, Binder binder, object[] args, System.Globalization.CultureInfo culture, object[] activationAttributes)
+        { 
+            if (assemblyName == null)
+                throw new ArgumentNullException(nameof(assemblyName));
+
+            return Activator.CreateInstance(assemblyName,
+                                            typeName,
+                                            ignoreCase,
+                                            bindingAttr,
+                                            binder,
+                                            args,
+                                            culture,
+                                            activationAttributes);
+        }
+
+        public ObjectHandle CreateInstance(string assemblyName, string typeName, object[] activationAttributes)
+        { 
+            if (assemblyName == null)
+                throw new ArgumentNullException(nameof(assemblyName));
+
+            return Activator.CreateInstance(assemblyName, typeName, activationAttributes);
+        }
+
+        public object CreateInstanceAndUnwrap(string assemblyName, string typeName)
+        {
+            ObjectHandle oh = CreateInstance(assemblyName, typeName);
+
```

Comment:
nit: extra line

Reply:
i added extraline in all methods...remove all?

Reply:
yes, it is neither accompanied by any type of  brace nor its more than a couple of lines

Reply:
is there a fixed rule for empty line? for example https://github.com/MarcoRossignoli/corefx/blob/ae8636faa74119668cdc605fd77a0a7361a31335/src/System.Runtime.Extensions/src/System/AppDomain.cs#L288 there isn't extraline

Reply:
I dont remember if there is coding guideline for this. This is the general feedback i got earlier.